packages/appstore/src/modules/trading-hub/index.tsx

```diff
@@ -1,7 +1,21 @@
 import * as React from 'react';
+import { useStores } from 'Stores';
+import { Authorize } from '@deriv/api-types';
+import WalletAccount from 'Components/wallet-account';
+import './trading-hub.scss';
 
 const TradingHub = () => {
-    return <div className='trading-hub'>Trading Hub</div>;
+    const { client } = useStores();
+
+    const wallet_accounts = (client.wallet_accounts || []) as Required<Authorize>['account_list'];
```

Comment:
@iman-fs Would be better to avoid type casting and add the typings in `packages/appstore/src/types/stores.types.ts` instead.

Reply:
changed it.